### PR TITLE
[T3A3][F10-A2]Lee Wenwei Johnervan

### PR DESCRIPTION
**Pull request by johnervan**
Ready For Review.

### CODE REVIEW

File: test/expected.txt [hunk 1]
```diff
@@ -8,7 +8,7 @@
 || Enter command: || [Command entered:  sfdfd]
 || add: Adds a person to the address book. Contact details can be marked private by prepending 'p' to the prefix.
 || Parameters: NAME [p]p/PHONE [p]e/EMAIL [p]a/ADDRESS  [t/TAG]...
-|| Example: add John Doe p/98765432 e/johnd@gmail.com a/311, Clementi Ave 2, #02-25 t/friends t/owesMoney
+|| Example: add John Doe p/98765432 e/johnd@gmail.com a/311, Clementi Ave 2, #02-25, 231534 t/friends t/owesMoney
```
Comment (Candiie) [inline test/expected.txt]: Well done on updating your tests!

File: doc/UserGuide.md [hunk 2]
```diff
@@ -43,10 +43,11 @@ Format: `add NAME [p]p/PHONE_NUMBER [p]e/EMAIL [p]a/ADDRESS [t/TAG]...`
 > be seen using the `viewall` command.
 > 
 > Persons can have any number of tags (including 0)
+> Person's ADDRESS has to containt the following in this specific format: BLOCK, STREET, UNIT, POSTAL CODE
 
 Examples: 
-* `add John Doe p/98765432 e/johnd@gmail.com a/John street, block 123, #01-01`
-* `add Betsy Crowe e/betsycrowe@gmail.com pp/1234567 pa/Newgate Prison t/criminal t/friend`
+* `add John Doe p/98765432 e/johnd@gmail.com a/John street, block 123, #01-01, 234123`
+* `add Betsy Crowe e/betsycrowe@gmail.com pp/1234567 pa/Newgate Prison, block 1, #02-01, 234343 t/criminal t/friend`
```
Comment (Candiie) [inline doc/UserGuide.md]: :+1:  Well done on updating user guide!

**Comment by Candiie** [comment]
Some comments added. Please close the PR after reading comments:

:+1: Hi Johnervan! Well done, simple and clean execution, however for this round of execution since this task is simple it is ok to do the changes and push all class changes into 1 commit however, in future commits do take note to commit after each significant changes instead of all in one :) (e.g. splitting the  classes implementations into different commits.)

:+1: again for the user guide + test update!

**Comment by johnervan** [comment]
Thanks!!

**Pull request closed without merging**